test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -667,6 +691,10 @@ if [[ ! -f "${KUBE_ROOT}/k8s_auth_data/kube-scheduler/kubeconfig" ]]; then
 	create-kubescheduler-kubeconfig
 fi
 
+ADDON_MANAGER_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+echo "${ADDON_MANAGER_TOKEN},system:addon-manager,uid:system:addon-manager,system:masters" >> "${KUBE_ROOT}/k8s_auth_data/known_tokens.csv"
```

Comment:
might change `uid:system:addon-manager` -> `admin` to conform to what configure-helper does?